docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -1951,6 +1951,86 @@ Return low-level information on the image `name`
        }
     }
 
+**Example response (Windows daemon)**:
+
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+
+    [
+        {
+            "Id": "sha256:105d76d0f40e38427c63023ffe649bf36fa85058d3469551e43e4dcc2431fb31",
+            "RepoTags": [
+                "microsoft/nanoserver:latest"
+            ],
+            "RepoDigests": [
+                "microsoft/nanoserver@sha256:aee7d4330fe3dc5987c808f647441c16ed2fa1c7d9c6ef49d6498e5c9860b50b"
+            ],
+            "Parent": "",
+            "Comment": "",
+            "Created": "2016-09-22T02:39:30.9154862-07:00",
+            "Container": "",
+            "ContainerConfig": {
+                "Hostname": "",
+                "Domainname": "",
+                "User": "",
+                "AttachStdin": false,
+                "AttachStdout": false,
+                "AttachStderr": false,
+                "Tty": false,
+                "OpenStdin": false,
+                "StdinOnce": false,
+                "Env": null,
+                "Cmd": null,
+                "Image": "",
+                "Volumes": null,
+                "WorkingDir": "",
+                "Entrypoint": null,
+                "OnBuild": null,
+                "Labels": null
+            },
+            "DockerVersion": "",
+            "Author": "",
+            "Config": {
+                "Hostname": "",
+                "Domainname": "",
+                "User": "",
+                "AttachStdin": false,
+                "AttachStdout": false,
+                "AttachStderr": false,
+                "Tty": false,
+                "OpenStdin": false,
+                "StdinOnce": false,
+                "Env": null,
+                "Cmd": [
+                    "c:\\windows\\system32\\cmd.exe"
+                ],
+                "Image": "",
+                "Volumes": null,
+                "WorkingDir": "",
+                "Entrypoint": null,
+                "OnBuild": null,
+                "Labels": null
+            },
+            "Architecture": "",
+            "Os": "windows",
+            "OsVersion": "10.0.14393",
```

Comment:
Yes. See my comment above: `Note currently the base images (up to and including 9D) only have the OS versions populated as a triple (such as 10.0.14393) rather than a quadtuple (such as 10.0.14393.206 for 9D). For future pushes of the base images, we'll populate the OSVersion field fully. From that point on, it will be able to use docker inspect to determine the version of the base image at the time it was docker pulled`